### PR TITLE
commit first pass at problem set and readme with deleted binder links

### PR DESCRIPTION
**Pull request by vaga3461**

### CODE REVIEW

**Comment by vaga3461** [comment]
binder link for reviewers:

https://mybinder.org/v2/gh/CUB-Computational-Tools/2020-ps-vaga3461/binder-python?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2F2020-ps-vaga3461%26branch%3Dweek4%26urlpath%3Dlab

File: week4.ipynb [hunk 1]
```diff
@@ -0,0 +1,402 @@
+{
```
Comment (MzCorona) [inline week4.ipynb:1]: <p>An intro before the learning goals... I like it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MzCorona) [inline week4.ipynb:1]: <p>First bullet point "discuss"... to what extent? Do you want them to be able to define/describe/list the factors? As a student, I wouldn't know what to make of "discuss" without some more clarity.</p><p> </p><p>Spelling of "calculations" in second sentence.</p><p> </p><p>What do you mean by "modularize"? As a student, I would find this scary to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (MzCorona) [inline week4.ipynb:1]: <p>spelling -- "interested", third line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (MzCorona) [inline week4.ipynb:1]: <p>is "grainsize" hyphenated or one-word?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (MzCorona) [inline week4.ipynb:1]: <p>I like that you define whether the input is a float, integer, boolean, etc. I also like the comments explaining what you wrote.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (MzCorona) [inline week4.ipynb:1]: <p>Maybe add a section where they put their answers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (MzCorona) [inline week4.ipynb:1]: <p>I would suggest a sentence or two that kind of wraps up the lesson. It helps students feel like they synthesized the information they learned here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (MzCorona) [inline week4.ipynb:1]: <p>Great job!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (vaga3461) [reply week4.ipynb:1]: I see it all different ways... one word, two words, or hyphenated. I'll go through and just try to make sure I'm being consistent everywhere - thanks!

**Pull request merged**